template_app_ipfire_by_zabbix_agent_active.xml

```diff
@@ -41,6 +41,9 @@ Created by Robin Roevens (robin.roevens (at) disroot.org)</description>
                 <template>
                     <name>Template Module IPFire Services by Zabbix Agent Active</name>
                 </template>
+                <template>
+                    <name>Template Module IPFire DHCP Clients by Zabbix Agent Active</name>
```

Comment:
* I'm not really sure if I want the dhcp leases metric in a separate template. I think a better place would be the current IPfire network stats template.
* The captive portal clients metric is independent of the dhcp leases, so that I think I would place that in the IPFire general template, since it is a bit of overkill to add an extra captive portal specific template for one single metric. If we would have more metrics about the captive portal, maybe then a new captive portal-specific template could be an option, but as I don't use captive portal myself, so I'm not sure if there are any more metrics that we could fetch there that could have any meaning.

Reply:
Fair enough. I don't have a strong preference. I do have a few other wish-list items I want to capture when I get time to add them but for these right now I think your suggestion is a good one. Thanks!

Reply:
You may always open issues with a feature request in it for each item you want captured.. I'm very interested in what the community would like to see monitored on IPFire. Maybe I can implement some of them myself if I find the time. If not, you can do so and add a pull request to the issue then.